src/glid.hpp

```diff
@@ -38,8 +38,7 @@ class Glid {
     GLuint id;
 
     // Required by Visual Studio to make it truly noncopyable
-    Glid(Glid&) /* = delete*/;
-    Glid& operator= (Glid&) /* = delete*/;
+    Glid(const Glid&) /* = delete*/;
+    Glid& operator= (const Glid&) /* = delete*/;
```

Comment:
I'm not sure, I think you would have added those as I don't use VS. I thought I had already 'fixed' that issue on master though. Still, it's definitely fixed now